tests/main/cgroup-tracking-failure/task.yaml

```diff
@@ -14,17 +16,28 @@ systems:
     - ubuntu-core-16-*
     - ubuntu-core-18-*
     - ubuntu-core-20-*  # tracking works correctly, for completeness
+
 prepare: |
     tests.cleanup prepare
     # This feature depends on the release-app-awareness feature
     snap set core experimental.refresh-app-awareness=true
     tests.cleanup defer snap unset core experimental.refresh-app-awareness
     "$TESTSTOOLS"/snaps-state install-local test-snapd-sh
     tests.cleanup defer snap remove --purge test-snapd-sh
+
 restore: |
     tests.cleanup restore
-    rm -f scenario.txt debug.txt cgroup.txt ps.txt run.txt
     systemctl --user stop dbus.service || true
+
+debug: |
+    set +e  # any of the files below may not exist and this causes the command to be indicated as failure
+    test -e scenario.txt && cat scenario.txt
```

Comment:
rather than do this, I would rather see us use `tests.cleanup defer rm scenario.txt` at the point in the test which we know that each file should exist, this test is also already preparing and restoring use `tests.cleanup prepare` etc so it shouldn't be a lot of work

Reply:
@anonymouse64 in this test we don't need to delete scenario.txt due to is it automatically deleted while the test is restored (after the debug). I think the problem here is when the test fails before the scenario.txt file is created, so it does not exist during debug phase.

Reply:
Ah I see I misread this test then

Reply:
@anonymouse64 np, thanks for the review